Formula/attr.rb

```diff
@@ -0,0 +1,26 @@
+class Attr < Formula
+  desc "Manipulate filesystem extended attributes"
+  homepage "https://savannah.nongnu.org/projects/attr"
+  url "https://mirror.csclub.uwaterloo.ca/nongnu/attr/attr-2.4.48.tar.gz"
+  sha256 "5ead72b358ec709ed00bbf7a9eaef1654baad937c001c044fe8b74c57f5324e7"
+  license "GPL-2.0-or-later"
+
+  depends_on "gettext" => :build
+  depends_on :linux
+
+  def install
+    # Fix No rule to make target `../libattr/libattr.la', needed by `attr'
```

Comment:
Has this been submitted upstream? If not: can it be, please.

Reply:
I ran a test build on linux and it looks like it's not needed anymore:
https://github.com/Homebrew/linuxbrew-core/pull/21359